base/download.jl

```diff
@@ -28,7 +28,7 @@ function find_curl()
         "/usr/bin/curl"
     elseif Sys.iswindows() && Sys.isexecutable(joinpath(get(ENV, "SYSTEMROOT", "C:\\Windows"), "System32\\curl.exe"))
         joinpath(get(ENV, "SYSTEMROOT", "C:\\Windows"), "System32\\curl.exe")
-    elseif Sys.which("curl") !== nothing
+    elseif !Sys.iswindows() && Sys.which("curl") !== nothing
```

Comment:
What's this unrelated looking change about?

Reply:
It's there since the times when #32412 was not merged and I was trying to get CI not to fail, but now that #32412 has been merged I don't know why that change is still shown. In fact on my local repository,
running
`git diff better-show master -- base/download.jl`
gives nothing.